docs/privacyidea.md

```diff
@@ -216,6 +216,41 @@ Use the following example:
         'excludeClientIPs'  => array("10.0.0.0-10.2.0.0", "192.168.178.2"),
     ),
 
+    /**
+     *  This filter is optional, if you want to disable clients by entityID, you should enable it.
+     */
+    22 => array(
+        'class'                => 'privacyidea:checkEntityID',
+
+        /**
+         *  Specify setPath and setKey to equal enabledPath and enabledKey
```

Comment:
The is *enabledPath and enabledKey from privacyidea:serverconfig* ?

Reply:
yes